src/main/java/com/google/api/generator/gapic/composer/ServiceClientClassComposer.java

```diff
@@ -108,8 +109,10 @@ public static ServiceClientClassComposer instance() {
     return INSTANCE;
   }
 
-  @Override
-  public GapicClass generate(Service service, Map<String, Message> messageTypes) {
+  public GapicClass generate(
+      Service service, Map<String, Message> messageTypes, Map<String, ResourceName> resourceNames) {
+    // TODO(miraleung): Clean up the hierarchy to avoid pass another parameter (resourceNames is
```

Comment:
Could we please move this TODO to the class level, since that's where the `implements` would go?

Reply:
Sure.